distribution/lib/Standard/Table/0.0.0-dev/src/Error.enso

```diff
@@ -67,3 +66,31 @@ type Input_Indices_Already_Matched (indices : [Integer])
 Input_Indices_Already_Matched.to_display_text : Text
 Input_Indices_Already_Matched.to_display_text =
     "The indices "+this.indices.short_display_text+" matched columns which have been matched earlier by other indices, so they did not introduce any new columns into the result."
+
+## Indicates that an aggregation calculation could not be completed.
+type Invalid_Aggregation (column:Text) (row:Integer) (message:Text)
+
+Invalid_Aggregation.to_display_text : Text
+Invalid_Aggregation.to_display_text =
+    "The "+this.column+" could not be calculated at "+this.row.to_text+" : "+this.message
+
+## Indicates that an floating point number was used in a grouping.
+type Floating_Point_Grouping (column:Text) (row:Integer)
+
+Floating_Point_Grouping.to_display_text : Text
+Floating_Point_Grouping.to_display_text =
+    "Grouping on floating points is not recommended within "+this.column+" at row "+this.row.to_text+"."
+
+## Indicates that a text value with a delimiter was included in a concatenation without any quote character
+type Unquoted_Delimiter (column:Text) (row:Integer)
+
+Unquoted_Delimiter.to_display_text : Text
+Unquoted_Delimiter.to_display_text =
+    "The "+this.column+" at row "+this.row.to_text+" contained an unquoted delimiter."
```

Comment:
I think some further explanation would be helpful. Because just looking at this, if I didn't discuss with you the behaviour of `Concat` in more detail, I wouldn't really know what is the issue here.

Wondering if it would make sense to have some optional `details : Text` field on problems which the GUI could show as a tooltip/foldable message which could include some practical insights into each warning. Not saying this should be done now, but I think it would be very helpful for the users at some point. But probably something to consider at a later polishing stage.

Reply:
Improved text but yes I think display and details on warnings will be a good thing when we get them to UI.